readme.MD

```diff
@@ -2,8 +2,12 @@
 
 RESTful API for bulletin board web application.
 
+[Checkout the UI Repository too!](https://github.com/romppane/Bulletin_Board_UI)
+
 ## Project configurating and launching:
 
+**Prerequisites:** **_Node_**:_10.16.1_, **_npm_**:_6.9.0_, **_Docker_**:_19.03.1_
```

Comment:
Make you make list out of these? Something like this

**Prerequisites:** 
- **_Node_**: _10.16.1_ 
- **_npm_**: _6.9.0_ 
- **_Docker_**: _19.03.1_